vimperator/install.rdf

```diff
@@ -15,7 +16,7 @@
         <em:targetApplication>
             <Description>
                 <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
-                <em:minVersion>38.0</em:minVersion>
+                <em:minVersion>45.0</em:minVersion>
```

Comment:
why update this value? Does it not work with version 38?
I know that are bugs but that doesn't mean it isn't compatible.